README.md

```diff
@@ -123,6 +135,7 @@ The following configuration values are available:
 | collectionsPrefix | redis | rest-storage:collections | The prefix for collections redis keys |
 | deltaResourcesPrefix | redis | delta:resources | The prefix for delta resources redis keys |
 | deltaEtagsPrefix | redis | delta:etags | The prefix for delta etags redis keys |
+| lockPrefix | redis | rest-storage:lock | The prefix for lock redis keys |
```

Comment:
I would suggest to set the default value of lockPrefix to **rest-storage:locks** since there will be multiple locked resources below this path.